docker-compose/monitor/Makefile

```diff
@@ -0,0 +1,50 @@
+.PHONY: build
+build: clean-jaeger
+	cd ../../ && \
+	make build-all-in-one && \
+	make docker-images-jaeger-backend && \
+	docker tag jaegertracing/all-in-one:latest jaegertracing/all-in-one:dev
+
+# run starts up the system required for SPM using the latest jaeger and otel images.
+.PHONY: run
+run: export JAEGER_IMAGE_TAG = latest
+run: _run-connector
+
+# run starts up the system required for SPM using the latest otel image and a development jaeger image.
+# Note: the jaeger "dev" image can be built with "make build".
+.PHONY: run-dev
+run-dev: export JAEGER_IMAGE_TAG = dev
+run-dev: _run-connector
+
+# _run-connector is the base target to bring up the system required for SPM using the new OTEL spanmetrics connector.
+.PHONY: _run-connector
+_run-connector: export OTEL_IMAGE_TAG = latest
+_run-connector: export OTEL_CONFIG_SRC = ./otel-collector-config-connector.yml
+_run-connector: export PROMETHEUS_QUERY_SUPPORT_SPANMETRICS_CONNECTOR = true
+_run-connector: export PROMETHEUS_QUERY_NAMESPACE = span_metrics
+_run-connector: export PROMETHEUS_QUERY_DURATION_UNIT = ms
+_run-connector:
+	docker compose -f docker-compose.yml up
+
+# The run-dev-processor make target is setup to allow devs to run the older spanmetrics processor setup, for example,
```

Comment:
```suggestion
# run the older spanmetrics processor setup, for example,
```

Reply:
Updated in: [f030268](https://github.com/jaegertracing/jaeger/pull/4452/commits/f030268105324ef71b6e89201e55c5959d732848).